### PR TITLE
Enable IDE0040: AddAccessibilityModifiers

### PR DESCRIPTION
**Pull request by xtqqczze**
https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0040

### CODE REVIEW

**Review by iSazonov — APPROVED** [review]

**Comment by iSazonov** [comment]
@xtqqczze Please resolve merge conflicts.

**Comment by xtqqczze** [comment]
Merge conflicts resolved.

**Review by iSazonov — APPROVED** [review]

File: test/tools/TestService/Program.cs [hunk 1]
```diff
@@ -5,9 +5,9 @@
 
 namespace TestService
 {
-    static class Program
+    internal static class Program
     {
-       static void Main()
+       private static void Main()
```
Comment (iSazonov) [inline test/tools/TestService/Program.cs:10]: I wonder why they are not public.

File: test/tools/TestExe/TestExe.cs [hunk 2]
```diff
@@ -7,9 +7,9 @@
 
 namespace TestExe
 {
-    class TestExe
+    internal class TestExe
     {
-        static int Main(string[] args)
+        private static int Main(string[] args)
```
Comment (iSazonov) [inline test/tools/TestExe/TestExe.cs:12]: Public?

File: test/tools/TestService/Program.cs (hunk 1 above)
Comment (xtqqczze) [reply test/tools/TestService/Program.cs:10]: This PR adds the default accessibility modifiers: `internal` for class and `private` for method.

It seems the class has no outside callers therefore accessibility is not important:

https://stackoverflow.com/questions/3110184/why-is-main-method-private

**Pull request merged**